### PR TITLE
Move program statistics to readme

### PR DESCRIPTION
**Pull request by Riaankl**
Fixes: #1000

### CODE REVIEW

**Review by nate-double-u — APPROVED** [review]
Thanks for this @Riaankl

File: README.md [hunk 1]
```diff
@@ -28,6 +28,31 @@ If you have any questions or need help getting started, please reach out though
...
+## Program Statistics
+
+These are the number of successful internships per year for each program.
+
+| Year | Program                          | Internships | Total (per year) |
+|------|----------------------------------|-------------|------------------|
+| 2023 | LFX Mentorship                   | TBD         | TBD              |
+|      | GSoC                             | TBD         |                  |
+| 2022 | LFX Mentorship                   | TBD         | TBD              |
+|      | GSoC                             | TBD         |                  |
```
Comment (aliok) [inline README.md:40]: @nate-double-u 
I see GSoC 2022 has 16 successful projects: https://summerofcode.withgoogle.com/archive/2022/organizations/cncf

Can you find out the number for LFX? I couldn't find such info publicly.

Comment (Riaankl) [reply README.md:40]: I just move the stats from a file to the README. We should propable open an issue to update the 2022 stats.

**Comment by Riaankl** [comment]
Check and there is an issue
https://github.com/cncf/mentoring/issues/910

File: README.md (hunk 1 above)
Comment (nate-double-u) [reply README.md:40]: I think https://github.com/cncf/mentoring/pull/1012 takes care of this.

**Pull request merged**